src/libraries/Microsoft.VisualBasic.Core/tests/FinancialTests.cs

```diff
@@ -13,7 +13,7 @@ public class FinancialTests
 
         // The accuracy to which we can validate some numeric test cases depends on the platform.
         private static readonly int s_precision = IsArmOrArm64OrAlpine ? 12 :
-            PlatformDetection.IsNetFramework ? 14 : 15;
+            (PlatformDetection.IsBrowser || PlatformDetection.IsNetFramework) ? 14 : 15;
```

Comment:
Just curious, this is due to a limitation in the libc that we use in browser?

Reply:
Yes.